.github/workflows/ci.yml

```diff
@@ -286,6 +286,48 @@ jobs:
           allowed-skips: test-fastapi  # TODO: drop once re-enabled
           jobs: ${{ toJSON(needs) }}
 
+  publish_next:
```

Comment:
@samuelcolvin The idea is for this job to only update the branch, and then `release` job would release both?